pandas/tests/io/excel/test_readers.py

```diff
@@ -1131,6 +1131,19 @@ def test_excel_high_surrogate(self, engine):
         actual = pd.read_excel("high_surrogate.xlsx")
         tm.assert_frame_equal(expected, actual)
 
+    def test_no_header_with_list_index_col(self, engine):
```

Comment:
Can you parametrize this with the `read_ext` fixture and change the extension rather than only testing .xlsx?